### DISCUSSION TITLE
Problem Reporting

### DISCUSSION THREAD

**Discussion by azadbolour** [azadbolour/eagerwords-community#2] [General]
Please report each problem in a new discussion under the General category. 

To help us better address any issues you run into, please include the following information in your problem reports if at all possible:

* The date and time of the problem.
* Detailed steps leading up to the problem.
* Text of any error reported by EagerWords.
* For the browser application, a copy of the logs reported in your browser console, your operating system and browser, and their versions.
* Screenshots where applicable.

**Comment by danconde** [comment]
3/6/2023
Open help topics
Playing with other Users: navigate to this page
Text: typo: othe -> other
Type: iOS

**Reply by azadbolour** [reply]
Thanks. Fix will be deployed in version 1.2 expected in April 2013.
